src/components/PivotTable/Droppable/Droppable.tsx

```diff
@@ -23,10 +24,7 @@ export interface DroppableProps<T> {
   /**
    * Map of all the keys to a display name of the key and a formatter and ordenator for the filter options
    */
-  keyMapping: Map<
-    keyof T,
-    { keyName: string; formatter?: (value: string) => string; ordenator?: (a: string, b: string) => number }
-  >
+  keyMap: KeyMap<T>
```

Comment:
I recommend to maintain as keyMapping because is the pattern 

```suggestion
  keyMapping: KeyMap<T>
```